express-zod-api/tests/__snapshots__/documentation-helpers.spec.ts.snap

```diff
@@ -1369,28 +1377,34 @@ exports[`Documentation helpers > depictString() > should set format, pattern and
 
 exports[`Documentation helpers > depictString() > should set format, pattern and min/maxLength props 13 1`] = `
 {
-  "format": "byte",
+  "contentEncoding": "base64",
+  "format": "base64",
```

Comment:
this turned out to be the recommended way in OAS 3.1
https://swagger.io/specification/#working-with-binary-data